app/policies/comment_policy.rb

```diff
@@ -0,0 +1,27 @@
+# app/policies/comment_policy.rb
+class CommentPolicy
+  attr_reader :user, :comment
+
+  def initialize(user, comment)
+    @user = user
+    @comment = comment
+  end
+
+  def destroy?
+    user == comment.author
+  end
+
+  def create?
+   true
+  end
+
+  def update?
+    user == @comment.author
```

Comment:
I'd stay consistent and use `comment.author` (without `@`) since that's what you used for `destroy?`.